### PR TITLE
More Prometheus ideas

### PR DESCRIPTION
**Pull request by gouthamve**
/cc @codesome @nikhita

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,6 +123,22 @@ Prometheus ideas:
...
+-	Issue: https://github.com/prometheus/prometheus/issues/11.
+
+#### Optimize queries using regex matchers for set lookups 
+
+-	Description: A common use case for regex matchers is to use them to query all series matching a set of label values, e.g. `up{instance=~"foo|bar|baz"}`. Grafana's template variables feature is a big user of that pattern. We could catch and split it into 3 different matchers, each selecting the three cases. This would make the templated queries produced by Grafana much faster.
+-	Recommended Skills: Golang
+-	Mentor(s): Goutham Veeramachaneni (@gouthamve)
+-	Issue: https://github.com/prometheus/prometheus/issues/2561.
+
+
```
Comment (nikhita) [inline README.md]: nit: remove the extra newlines? :)

**Comment by codesome** [comment]
:+1:

File: README.md [hunk 2]
```diff
@@ -123,6 +123,22 @@ Prometheus ideas:
...
+-	Recommended Skills: Golang
+-	Mentor(s): Ganesh Vernekar (@codesome), Goutham Veeramachaneni (@gouthamve)
+-	Issue: https://github.com/prometheus/prometheus/issues/11.
+
+#### Optimize queries using regex matchers for set lookups 
+
+-	Description: A common use case for regex matchers is to use them to query all series matching a set of label values, e.g. `up{instance=~"foo|bar|baz"}`. Grafana's template variables feature is a big user of that pattern. We could catch and split it into 3 different matchers, each selecting the three cases. This would make the templated queries produced by Grafana much faster.
+-	Recommended Skills: Golang
+-	Mentor(s): Goutham Veeramachaneni (@gouthamve)
+-	Issue: https://github.com/prometheus/prometheus/issues/2561.
```
Comment (codesome) [inline README.md]:
```suggestion
-	Issue: https://github.com/prometheus/prometheus/issues/2651.
```

File: README.md (hunk 1 above)
Comment (gouthamve) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (gouthamve) [reply README.md]: Done

**Pull request merged**